core-common/src/main/java/org/zalando/nakadi/enrichment/Enrichment.java

```diff
@@ -39,10 +39,12 @@ public void validate(final EventTypeBase eventType) throws InvalidEventTypeExcep
         }
     }
 
-    public void enrich(final BatchItem batchItem, final EventType eventType) throws EnrichmentException {
+    public void enrich(final BatchItem batchItem, final EventType eventType, final String schemaVersion)
+            throws EnrichmentException {
+
         for (final EnrichmentStrategyDescriptor descriptor : eventType.getEnrichmentStrategies()) {
             final EnrichmentStrategy strategy = getStrategy(descriptor);
-            strategy.enrich(batchItem, eventType);
+            strategy.enrich(batchItem, eventType, schemaVersion);
```

Comment:
it would be better to pass a pair event type -> schema version, that will prevent errors like schema version does not exist for such event type